src/widgets/principles/ui/principle-card/principle-card.module.scss

```diff
@@ -20,7 +17,23 @@
   border: 1px solid rgb(255 219 32 / 8%);
   border-radius: 12px;
 
-  & .card-header {
+  &::after {
+    content: '';
+
+    position: absolute;
+    z-index: 1;
```

Comment:
Do we really need all those `z-indexes`?

Reply:
fixed. The only `z-index` I left, because of the yellow `blob` is overlayed the text

before:
<img width="500" alt="Screenshot 2024-11-10 at 5 33 40 PM" src="https://github.com/user-attachments/assets/be3bf400-af90-46db-a25c-77b28382978a">

after:
<img width="500" alt="Screenshot 2024-11-10 at 5 33 30 PM" src="https://github.com/user-attachments/assets/9e5799f8-5e8e-472b-8ec8-6b4a020f24d1">